gui/system/forms.py

```diff
@@ -2510,9 +2510,22 @@ class CertificateAuthorityCreateInternalForm(MiddlewareModelForm, ModelForm):
         required=True,
         help_text=models.CertificateAuthority._meta.get_field('cert_name').help_text
     )
+    cert_key_type = forms.ChoiceField(
+        label=_('Key Type'),
+        help_text=_('Please select a key type to proceed with'),
```

Comment:
This help text just restates the field name.  What are the key types?  Which is preferred?  Is there a link that explains it?

Reply:
The key types are RSA or EC which would be shown to the user in the drop down. There are advantages/disadvantages to using both. Both have their use cases and strengths. We default to using RSA. Here's a link explaining a bit more about these - https://crypto.stackexchange.com/questions/1190/why-is-elliptic-curve-cryptography-not-widely-used-compared-to-rsa. Please let me know how you think we should move with this

Reply:
`` 'See https://crypto.stackexchange.com/questions/1190/why-is-elliptic-curve-cryptography-not-widely-used-compared-to-rsa for more information about key types.' ``